Update 404.html

garaynakca opened the pull request:
updated headers

<!-- readthedocs-preview civicactions-handbook start -->
----
:books: Documentation preview :books:: https://civicactions-handbook--1087.org.readthedocs.build/en/1087/

<!-- readthedocs-preview civicactions-handbook end -->

garaynakca committed:
Update 404.html

updated headers

dmundra commented on .config/theme_overrides/404.html:
```diff
@@ -21,8 +21,9 @@
+<h1>404</h1>
+<div class="leadtext">We looked, but couldn’t quite find that page.</div>
+
 <h2>
```
@garaynakca I had to load it locally to see the changes:

![Screenshot of 404 page](https://user-images.githubusercontent.com/7096681/227343075-4a09ba21-36ed-4c11-a8c8-920c0ecd1f9f.png)

Can you change the `<h2>` to plain `<div>` as well? Also I think in another branch you are reducing the size of the leadtext?

ContentBrewmaster replied:
@dmundra I think Gretchen fixed the h-tag issue in patch-3

dmundra replied:
Ah I see it now. I merged it in.

garaynakca committed:
Update 404.html (#1098)

* Update 404.html

Changed h2 to div text

* [pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

---------

Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>
